### PR TITLE
liberation fonts: updates

### PR DESCRIPTION
**Pull request by dtzWill**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/data/fonts/redhat-liberation-fonts/default.nix [hunk 1]
```diff
@@ -50,36 +50,36 @@ let
 
 in {
   liberation_ttf_v1_from_source = common rec {
-    version = "1.07.4";
-    url = "https://releases.pagure.org/liberation-fonts/liberation-fonts-${version}.tar.gz";
-    sha256 = "01jlg88q2s6by7qv6fmnrlx0lwjarrjrpxv811zjz6f2im4vg65d";
+    version = "1.07.5";
+    url = https://github.com/liberationfonts/liberation-1.7-fonts/files/2175696/liberation-fonts-1.07.5.tar.gz;
```
Comment (volth) [inline pkgs/data/fonts/redhat-liberation-fonts/default.nix]: maybe ```fetchFromGitHub { ...; rev = version; }``` for source versions ?
Repos have release tags (https://github.com/liberationfonts/liberation-1.7-fonts/releases https://github.com/liberationfonts/liberation-fonts/releases)

**Comment by worldofpeace** [comment]
@volth Any reason why we need binary and source packages?

**Comment by volth** [comment]
> @volth Any reason why we need binary and source packages?

Binary could be removed

**Comment by worldofpeace** [comment]
> > @volth Any reason why we need binary and source packages?
> 
> Binary could be removed

Did the thing https://github.com/NixOS/nixpkgs/pull/51331/commits/0e822cb345bc79bed67af4fbd041455a073ffa23

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -15382,11 +15382,10 @@ with pkgs;
...
 
   inherit (callPackages ../data/fonts/redhat-liberation-fonts { })
-    liberation_ttf_v1_from_source
-    liberation_ttf_v1_binary
-    liberation_ttf_v2_from_source
-    liberation_ttf_v2_binary;
-  liberation_ttf = liberation_ttf_v2_binary;
+    liberation_ttf_v1
+    liberation_ttf_v2
+    ;
```
Comment (worldofpeace) [inline pkgs/top-level/all-packages.nix]: The placement of this semicolon is purposeful.

**Pull request closed without merging**